editor/animation_track_editor.cpp

```diff
@@ -7022,6 +7026,10 @@ bool AnimationTrackEditor::is_grouping_tracks() {
 	return !view_group->is_pressed();
 }
 
+bool AnimationTrackEditor::is_function_name_pressed() {
+	return function_name_toggler->is_pressed();
```

Comment:
If pressed state is blue icon, it would appear that it is valid, so it would be clearer to just toggle between gray and light colors.

Reply:
Sorry, but I didn't understand it very well.  Should gray indicate that it isn't active (i.e, the names are hidden), and a lighter color to indicate that the names are shown? If that is the case, is there a problem using the current blue tint? Since I'll just have to invert the two.

Reply:
Should make the following state:

- Function name shown: White icon
- Function name hidden: Gray icon

Then blue color should not be used here, so you will need to control the button color yourself, either by overriding the default behavior of the button or by overriding the theme.